automated-comments/go/hamming/extra_if_strings_empty.md

```diff
@@ -0,0 +1,2 @@
+The extra check if the `strings` are empty is not wrong but does not add value either. It would make the function
+more readable if it was removed.
```

Comment:
I prefer the passive voice to the imperative case here to make it more friendly. So "This can be simplified by removing..." rather than "Simplify by removing...".

I think it:
- Feels friendlier
- Makes someone think about what needs doing, rather than following an instruction.

If @kytrinyx agrees, can this get added to the guidance please, @SleeplessByte.

Reply:
Certainly. Personally I'm not a fan of the passive voice but I realise this is probably just cultural difference. In general, if you tell a Dutch person (in my experience) that they _can_ or _could_ do something, they'll probably respond with something like "ok thnx, I'll leave it".

I don't have a good solution 😓

Reply:
Yeah, I also agree. But as these comments are for approval, then we are giving people the choice of doing it or not. I don't think we should be giving people strong instructions when we're also saying "This is fundamentally fine". 

I would suggest "You can simplify..." but but I agree that we want to avoid "You".

Reply:
Yeah. I think in _this particular case_ "This can be simplified by removing ..." works well. 

This is not easy 😅

Reply:
adjusted to `This can be simplified by removing`. I think it is more gentle.

Reply:
I agree! (Sorry, I completely missed this as it was in a collapsed section in the PR).